packages/@aws-cdk/aws-route53/lib/util.ts

```diff
@@ -49,7 +49,7 @@ class ValidationError extends Error {
  *        <li>Otherwise, append +.+, +zoneName+ and a trailing +.+</li>
  *      </ul>
  */
-export function determineFullyQualifiedDomainName(providedName: string, hostedZone: IHostedZone): string {
+export function determineFullyQualifiedDomainName(providedName: string, hostedZone: IHostedZone, removeDomainNamePeriodEnd?: boolean): string {
```

Comment:
This function is interesting.

Is there any scenario the `removeDomainNamePeriodEnd` should be false with the imported HostedZone?